qiskit/visualization/circuit/circuit_visualization.py

```diff
@@ -28,30 +28,40 @@
 
 import logging
 import os
+import shutil
 import subprocess
 import tempfile
-import shutil
 import typing
 from warnings import warn
 
 from qiskit import user_config
-from qiskit.utils import optionals as _optionals
 from qiskit.circuit import ControlFlowOp, Measure
+from qiskit.utils import deprecate_arg
+from qiskit.utils import optionals as _optionals
+
+from ..exceptions import VisualizationError
+from ..utils import _trim as trim_image
+from . import _utils
 from . import latex as _latex
-from . import text as _text
 from . import matplotlib as _matplotlib
-from . import _utils
-from ..utils import _trim as trim_image
-from ..exceptions import VisualizationError
+from . import text as _text
 
 if typing.TYPE_CHECKING:
     from typing import Any
+
     from qiskit.circuit import QuantumCircuit  # pylint: disable=cyclic-import
 
 
 logger = logging.getLogger(__name__)
 
 
+@deprecate_arg(
+    "justify",
+    since="1.1.0",
+    deprecation_description="Setting circuit_drawer()’s justify argument to a value other than "
+    "'left', 'right', or 'none'. Until deprecation, default 'left' will be used instead.",
+    predicate=lambda justify: justify not in ["left", "right", "none"],
```

Comment:
You'll need to import the function

```suggestion
    predicate=_is_valid_justify_arg,
```